application/src/test/java/org/opentripplanner/updater/GtfsRealtimeFuzzyTripMatcherTest.java

```diff
@@ -2,47 +2,146 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.opentripplanner.transit.model._data.TimetableRepositoryForTest.id;
 
 import com.google.transit.realtime.GtfsRealtime.TripDescriptor;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.function.Function;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.opentripplanner.GtfsTest;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.opentripplanner.graph_builder.issue.api.DataImportIssueStore;
+import org.opentripplanner.model.calendar.CalendarServiceData;
+import org.opentripplanner.transit.model._data.TimetableRepositoryForTest;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.network.Route;
+import org.opentripplanner.transit.model.network.TripPattern;
+import org.opentripplanner.transit.model.site.RegularStop;
+import org.opentripplanner.transit.model.timetable.RealTimeTripTimes;
+import org.opentripplanner.transit.model.timetable.Trip;
+import org.opentripplanner.transit.model.timetable.TripTimesFactory;
 import org.opentripplanner.transit.service.DefaultTransitService;
+import org.opentripplanner.transit.service.SiteRepository;
+import org.opentripplanner.transit.service.SiteRepositoryBuilder;
+import org.opentripplanner.transit.service.TimetableRepository;
 
-public class GtfsRealtimeFuzzyTripMatcherTest extends GtfsTest {
+public class GtfsRealtimeFuzzyTripMatcherTest {
+
+  private static final String ROUTE_ID = "r1";
+  private static final String FEED_ID = TimetableRepositoryForTest.FEED_ID;
+  private static final LocalDate SERVICE_DATE = LocalDate.of(2024, 11, 13);
+  private static final String GTFS_SERVICE_DATE = SERVICE_DATE.toString().replaceAll("-", "");
+  private static final int SERVICE_CODE = 555;
+  private static final SiteRepositoryBuilder siteRepositoryBuilder = SiteRepository.of();
+  private static final TimetableRepositoryForTest TEST_MODEL = new TimetableRepositoryForTest(
+    siteRepositoryBuilder
+  );
+  private static final RegularStop STOP_1 = TEST_MODEL.stop("s1").build();
+  private static final RegularStop STOP_2 = TEST_MODEL.stop("s2").build();
+  private static final TimetableRepository TT_REPO = new TimetableRepository(
```

Comment:
I think it would be more clear if you just wrote TIMETABLE_REPOSITORY and it's not too long. Had to think for like 10 seconds what TT means (thought it was trip times at first) when I encountered this variable later in code.

Reply:
Done in 2be774b24293b222a92b63d5476f9d06ee2e494b